# I don't care enough to look into what caused these errors, so this is just superficial fixing.
# I also patch out the credits print, because it makes vim open a new buffer when doing :make
diff -Nurp armips-old/CMakeLists.txt armips/CMakeLists.txt
--- armips-old/CMakeLists.txt	2015-05-25 08:21:28.203183885 +0200
+++ armips/CMakeLists.txt	2015-05-25 08:21:45.112972483 +0200
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8)
 project(armips) 
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -fpermissive")
 
 include_directories(.)
 
@@ -24,8 +24,6 @@ add_executable(armips
 	Util/FileClasses.h
 	Util/ByteArray.cpp
 	Util/ByteArray.h
-	Util/StringFormat.cpp
-	Util/StringFormat.h
 	
 	Core/ELF/ElfTypes.h
 	Core/ELF/ElfRelocator.cpp
@@ -46,8 +44,6 @@ add_executable(armips
 	Core/Common.h
 	Core/Misc.cpp
 	Core/Misc.h
-	Core/MathParser.cpp
-	Core/MathParser.h
 	Core/Directives.cpp
 	Core/Directives.h
 	Core/Assembler.cpp
diff -Nurp armips-old/Main/Tests.h armips/Main/Tests.h
--- armips-old/Main/Tests.h	2015-05-25 08:21:28.210183797 +0200
+++ armips/Main/Tests.h	2015-05-25 08:21:45.112972483 +0200
@@ -1,6 +1,5 @@
 #pragma once
 #include "Util/Util.h"
-#include <Windows.h>
 
 class TestRunner
 {
diff -Nurp armips-old/Main/main.cpp armips/Main/main.cpp
--- armips-old/Main/main.cpp	2015-05-25 08:21:28.210183797 +0200
+++ armips/Main/main.cpp	2015-05-25 08:22:33.659365570 +0200
@@ -25,7 +25,6 @@ int wmain(int argc, wchar_t* argv[])
 		return !runTests(argv[1]);
 #endif
 
-	Logger::printLine(ARMIPSNAME " Assembler v0.7d (" __DATE__ " " __TIME__ ") by Kingcom");
 	StringList arguments = getStringListFromArray(argv,argc);
 
 	if (arguments.size() < 2)
